services/user/TokenSys/ui/src/App.tsx

```diff
@@ -76,6 +83,7 @@ function App() {
         try {
             await transfer(data);
             reset();
+            invalidateSharedBalancesQuery();
```

Comment:
This causes the query above to refetch...but not the instance of that query being consumed by `shared-balances.tsx` so it has no effect.

Reply:
I would still call this here (given my proposal above), but I do recommend moving it down a few lines next to `invalidateTransferHistoryQuery();`.